### DISCUSSION TITLE
请求支持兼容Hysteria2协议

### DISCUSSION THREAD

**Discussion by pooneyy** [v2rayA/v2rayA#1508] [General]
请求支持兼容Hysteria2协议

**Comment by EiPiFun** [comment]
v2ray supports Hysteria2 now, please notify me if any updates come:)

**Reply by pooneyy** [reply]
> v2ray supports Hysteria2 now, please notify me if any updates come:)

When pulling subscriptions, the Hysteria2 node still cannot be pulled. (current version 2.2.7.1)
拉取订阅的时候，仍然拉取不到 Hysteria2 的节点
